pytket/qir/conversion/conversion.py

```diff
@@ -202,6 +202,19 @@ def __init__(
             ),
         )
 
+        # void mz_to_reg(qubit, i1* creg, int)
```

Comment:
Could you write the signature here in full? What's the `int` type referring to?

Reply:
done